src/libs/ajax.js

```diff
@@ -1103,12 +1101,13 @@ const Metrics = signal => ({
         appPath: Nav.getCurrentRoute().name
       }
     }
+
     // Remove the metricsEnabled feature flag once TOS and all metrics projects are setup
-    return metricsEnabled && fetchBard('api/event', _.mergeAll([authOpts(), jsonBody(body), { signal, method: 'POST' }]))
+    fetchBard('api/event', _.mergeAll([authOpts(), jsonBody(body), { signal, method: 'POST' }]))
   }),
 
   syncProfile: withErrorIgnoring(() => {
-    return metricsEnabled && fetchBard('api/syncProfile', _.merge(authOpts(), { signal, method: 'POST' }))
+    fetchBard('api/syncProfile', _.merge(authOpts(), { signal, method: 'POST' }))
```

Comment:
We should `return` the promise to keep everything linked up properly, even though we're currently ignoring errors.